lib/AST/MicrosoftMangle.cpp

```diff
@@ -2131,9 +2131,12 @@ void MicrosoftCXXNameMangler::mangleCallingConvention(CallingConv CC) {
 
   switch (CC) {
     default:
-      llvm_unreachable("Unsupported CC for mangling");
+      llvm::errs() << "Unsupported CC for mangling: " << CC << ".\n";
     case CC_Win64:
     case CC_X86_64SysV:
+    // FIXME: SwiftCC should have its own mangling specifier.
+    // For now, don't do anything special for SwiftCC mangling.
+    case CC_Swift:
```

Comment:
This will add `A` to the mangling.

Reply:
Is there a better option from the preexisting symbols? The idea here is just to treat SwiftCC like any other standard calling convention. What would you suggest?

Reply:
I think A is fine for now.  But (1) this should go in LLVM, not in our private fork, and (2) when you bring it up there, someone might point out a friendlier way of doing it.